package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@redpanda-data/docs-extensions-and-macros",
-  "version": "3.2.7",
+  "version": "3.2.9",
```

Comment:
shouldn't this be 8? xD

Reply:
Ah I see that https://github.com/redpanda-data/docs-extensions-and-macros/pull/40/files is 8